File: pbweberogwcl/src/app/dati-progetto/commons/requests/modifica-sede-intervento-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { SedeProgettoDTO } from '../dto/sede-progetto-dto';
export interface ModificaSedeInterventoRequest {
  idSedeInterventoAttuale?: number;
  sedeIntervento?: SedeProgettoDTO;
}

File: pbweberog/pbweberog-jar/src/main/java/it/csi/pbandi/pbweberog/integration/request/ModificaSedeInterventoRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweberog.integration.request;

import it.csi.pbandi.pbweberog.dto.datiprogetto.SedeProgettoDTO;

public class ModificaSedeInterventoRequest {
	private SedeProgettoDTO sedeIntervento;
	private Long idSedeInterventoAttuale;
	public SedeProgettoDTO getSedeIntervento() {
		return sedeIntervento;
	}
	public void setSedeIntervento(SedeProgettoDTO sedeIntervento) {
		this.sedeIntervento = sedeIntervento;
	}
	public Long getIdSedeInterventoAttuale() {
		return idSedeInterventoAttuale;
	}
	public void setIdSedeInterventoAttuale(Long idSedeInterventoAttuale) {
		this.idSedeInterventoAttuale = idSedeInterventoAttuale;
	}
	
	
	
}
